packages/koa/README.md

```diff
@@ -50,6 +50,12 @@ app.on("error", (error) => {
 });
 ```
 
+Note that generator-based middleware are deprecated in Koa version 2.x, and support for them will be removed in the next major Koa version.
+
+Our instrumentation does not instrument generator-based middleware. As described in [the Koa 2.x migration guide][guide], you can use the `koa-convert` library to convert them to new-style async middleware.
```

Comment:
Looks good, some very minor changes:

Note that generator-based middleware was deprecated in Koa version 2.x, and the next major Koa version will remove support for them entirely. 

Our instrumentation does not instrument generator-based middleware. Our [Koa 2.x migration guide][guide] explains how you can use the `koa-convert` library to convert them to new-style async middleware.

Reply:
Thank you! Applied with a minor change as discussed ("Our" -> "The")